### ISSUE TITLE
[release 0.10] issue NXView if opengl missing

### ISSUE THREAD

**Issue by payno** [silx-kit/silx#2465]
```
======================================================================
ERROR: test_replace_view_in_composite (silx.gui.data.test.test_dataviewer.TestDataViewer)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/payno/Documents/dev/release/silx_0_10/envpy_3_5/lib/python3.5/site-packages/silx/gui/data/test/test_dataviewer.py", line 244, in test_replace_view_in_composite
    [v.modeId() for v in nxdata_view.availableViews()])
AttributeError: 'NoneType' object has no attribute 'availableViews'

======================================================================
ERROR: test_replace_view_in_composite (silx.gui.data.test.test_dataviewer.TestDataViewerFrame)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/payno/Documents/dev/release/silx_0_10/envpy_3_5/lib/python3.5/site-packages/silx/gui/data/test/test_dataviewer.py", line 244, in test_replace_view_in_composite
    [v.modeId() for v in nxdata_view.availableViews()])
AttributeError: 'NoneType' object has no attribute 'availableViews'
```

**Comment by payno** [comment]
should be fixed by Pr #2466

**Comment by payno** [comment]
fix it
